README.adoc

```diff
@@ -0,0 +1,85 @@
+:imagesdir: ./screenshots
+
+= Visualize Sync Metrics Data on Openshift
+
+== Overview
+
+This guide will go through the steps to help you setup Influxdb and Grafana on Openshift to receive metrics data from the RHMAP sync framework, and view them in a Grafana dashboard. At the end, you should have a dashboard look like this:
+
+image::sync-stats-dashboard.png[]
+
+== Prerequisites
+
+You should have an Openshift cluster running somewhere. Since this guide will use persistent volumes to persist data, please make sure there are some of them available on the cluster. An link:pvs-example.yaml[example] file is provided in this repo to help with that.
+
+You should also have the https://docs.openshift.org/latest/cli_reference/get_started_cli.html[Openshift command line tool (oc)] installed locally and target to the given Openshift cluster.
+
+You should also add the cluster as an environment on RHMAP and can deploy a cloud app to the cluster.
```

Comment:
I think this could be clearer, maybe:

```
You should also add the cluster as an environment on RHMAP and be able to deploy cloud apps to the cluster.
```